client/src/components/Routes.js

```diff
@@ -15,6 +17,8 @@ const Routes = () => (
       < Route exact path="/users/new" component={SignUp} />
       < Route exact path="/user-sessions/new" component={SignIn} />
       < AuthenticatedRoute exact path="/home" component={Home} />
+      < AuthenticatedRoute exact path="/warm-fuzzies/new" component={Give} />
```

Comment:
we should probably call this `NewWarmFuzzy`, `AwardWarmFuzzy` or `GiveWarmFuzzy` to indicate what "noun" we're working with

Reply:
Updated in later PR